.github/workflows/release.yaml

```diff
@@ -108,12 +108,22 @@ jobs:
   build-python-wheels:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: true
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [macos-latest, windows-latest]
+        include:
+          - os: ubuntu-latest
+            target: x86_64
+          - os: ubuntu-latest
+            target: aarch64
+          - os: ubuntu-latest
+            target: source
```

Comment:
FWIW I'm not personally familiar with how these work — is `source` an alternative to `aarch64` — maybe wheels can either contain architecture-specific code or just the python source code? Great if so (and even if not, no need to explain it to me! Though if others are confused, adding a comment would be helpful)

Reply:
I see — this goes to an sdist, which is the source code. (I had seen how we were using it, just was less familiar with why the distinction from the python side). Thank you!